go/ephemeral/device_ek.go

```diff
@@ -138,12 +144,11 @@ func signAndPublishDeviceEK(ctx context.Context, g *libkb.GlobalContext, generat
 }
 
 type deviceEKsResponse struct {
-	Results []struct {
-		MerklePayload string `json:"merkle_payload"`
-		Sig           string `json:"sig"`
-	} `json:"results"`
+	Sigs []string `json:"sigs"`
 }
 
+// GetActiveDeviceEKMetadata fetches the latest deviceEK for each of your
+// devices, filtering out the ones that are stale.
```

Comment:
pre-existing bug. We should split this function into  `maybeStale` and `active` like in user_ek -- the `getServerMaxDeviceEK` is busted if the latest key is stale.